### DISCUSSION TITLE
Thanks to Patrick

### DISCUSSION THREAD

**Discussion by rogercreagh** [TechbeeAT/jtxBoard#294] [General]
I've been meaning to post for a while to thank Patrick for his work in producing an excellent and very useful app. The icing on the cake for me was the introduction of the hi-contrast e-ink theme which has really made it sparkle on my Boox notepad.

I use jtxB daily - only for a journal, and very occasionally for notes (I'm too disorganised to ever do more than start to-do lists and then ignore them).

The thing now is to try and avoid bloat and feature creep - I've more or less stopped suggesting ideas as the app performs its main functions well for me. I am starting to wonder whether sync is starting to slow down now my journal has almost 200 entries...

**Comment by patrickunterwegs** [comment]
Thanks @rogercreagh for your feedback 😃 

Yeah, feature bloat might become a problem, but I think as long as the UI stays more or less clear, additional features are still acceptable. And then there are proposals like integrating the Android calendar storage, this is a whole other level then 😉 

About the 200+ entries I'll review this and check how the app performs. But do you mean the sync is slow or the app scrolling in the list gets slow when you have many entries visible?
